programs/helium-sub-daos/src/instructions/issue_rewards_v0.rs

```diff
@@ -107,57 +152,59 @@ pub fn handler(ctx: Context<IssueRewardsV0>, args: IssueRewardsArgsV0) -> Result
     .try_into()
     .unwrap();
 
+  let total_emissions = ctx
+    .accounts
+    .sub_dao
+    .emission_schedule
+    .get_emissions_at(ctx.accounts.clock.unix_timestamp)
+    .unwrap();
+
+  mint_v0(
+    ctx.accounts.mint_dnt_emissions_ctx().with_signer(&[&[
+      b"sub_dao",
+      ctx.accounts.dnt_mint.key().as_ref(),
+      &[ctx.accounts.sub_dao.bump_seed],
+    ]]),
+    MintArgsV0 {
+      amount: total_emissions
+        .checked_div(100.try_into().unwrap())
+        .unwrap()
+        .checked_mul(94)
```

Comment:
Ordering matters here, right? Don't we lose precision by dividing by 100 first? Should do it after the mul.

Reply:
Yep definitely, fixed